2-mandatory/3-shopping-list.js

```diff
@@ -20,6 +20,25 @@ let pantry = {
 
 function createShoppingList(recipe) {
   // write code here
+  let shoppingFor = {};
+  let shoppingList = [];
+  // shoppingList= pantryItems.filter(item => !recipe["ingredients"].includes(item))
```

Comment:
Well done solving the task! Why comment out these two lines in preference for the `for loop`?